src/app/pages/subscriptions/confirm-subscription-wrapper.jsx

```diff
@@ -1,8 +1,9 @@
 import React from 'react';
 import { string } from 'prop-types';
-
+import {
+  Box,
```

Comment:
This should have given you a bunch of warnings. I'm not sure of the specific language but:

- Box isn't used
- The imports from `prop-types` and `./confirm-subscription` shouldn't be in the same group
- Neither should the ChplConfirmSubscription and the AppWrapper